rules-tests/Orm30/Rector/MethodCall/CastDoctrineExprToStringRector/Fixture/no-change.php.inc

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Rector\Doctrine\Tests\Orm30\Rector\MethodCall\CastDoctrineExprToStringRector\Fixture;
+
+use Doctrine\ORM\Query\Expr;
+
+final class ExprBuilderCall
```

Comment:
use different name to avoid class conflict

```suggestion
final class SkipLiteral
```

Reply:
Updated class name